cmd2/cmd2.py

```diff
@@ -37,14 +37,17 @@
 import re
 import sys
 import threading
+import types
 from code import InteractiveConsole
 from collections import namedtuple
 from contextlib import redirect_stdout
-from typing import Any, Callable, Dict, Iterable, List, Mapping, Optional, Tuple, Type, Union
+from typing import Any, AnyStr, Callable, Dict, Iterable, List, Mapping, Optional, Tuple, Type, Union
 
 from . import ansi, constants, plugin, utils
 from .argparse_custom import DEFAULT_ARGUMENT_PARSER, CompletionItem
 from .clipboard import can_clip, get_paste_buffer, write_to_paste_buffer
+from .command_definition import _REGISTERED_COMMANDS, CommandSet, _partial_passthru
```

Comment:
importing a couple protected members feels off

Reply:
I made them protected to indicate to external users that they shouldn't directly access these. It's for internal cmd2 use only.